dotCMS/src/main/java/com/dotmarketing/business/ChainableCacheAdministratorImpl.java

```diff
@@ -134,10 +134,10 @@ public void setCluster(Server localServer) throws Exception {
 				}
 	            localServer = Server.builder(localServer).withCachePort(Integer.parseInt(bindPort)).build();
    
-                List<String> myself = new ArrayList<String>();
+                List<String> myself = new ArrayList<>();
                 myself.add(localServer.getServerId());
 
-                List<Server> aliveServers = serverAPI.getAliveServers();
+                List<Server> aliveServers = serverAPI.getAliveServers(myself);
```

Comment:
Let's replace the myself list with an inline usage of `Collections.singletonList(localServer.getServerId())`